server/api/roster/roster.controller.ts

```diff
@@ -31,6 +31,44 @@ import { CustomColumnValue, RosterColumnInfo, RosterColumnType } from './roster.
 import { UserRole } from '../user/user-role.model';
 import { ChangeType, RosterHistory } from './roster-history.model';
 import { baseRosterColumns } from './roster-entity';
+import { Role } from '../role/role.model';
+
+export async function addRosterEntry(org: Org, role: Role, rosterEntryData: RosterEntryData) {
```

Comment:
Can you move this to `/server/util/roster-utils.ts`? It also needs to take a `manager` as mentioned above so that it can be used inside transactions.